.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Sublime Text version [e.g. 4085]
+- LSP version [e.g. 1.0.12]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
Templates often use HTML comments (`<!-- -->` for those instructions that are supposed to be removed when filing. Otherwise, if the user forgets to remove them, they'll end up in the report.

Reply:
Hmm... these HTML comments always confuse me when I'm writing an issue. I think it's okay to not have them.